Libraries/LibWeb/CSS/Parser/PropertyParsing.cpp

```diff
@@ -2622,7 +2622,15 @@ RefPtr<CSSStyleValue const> Parser::parse_font_language_override_value(TokenStre
             dbgln_if(CSS_PARSER_DEBUG, "CSSParser: Failed to parse font-language-override: unexpected trailing tokens");
             return nullptr;
         }
-        auto length = string_value.code_points().length();
+        auto length = string_value.bytes().size();
+        if (length == 0) {
+            dbgln_if(CSS_PARSER_DEBUG, "CSSParser: Failed to parse font-language-override: <string> value is empty");
+            return nullptr;
+        }
+        if (!all_of(string_value.bytes(), is_ascii)) {
```

Comment:
There should be an is_ascii() helper on String we can use here

Reply:
I've added an `is_ascii()` method to `FlyString` and am now using that.